Platforms/OSX/AppKit/NSApplication.swift

```diff
@@ -287,7 +287,7 @@ class NSApplication : NSResponder, NSUserInterfaceValidations, NSAccessibilityEl
   @available(OSX 10.10, *)
   func accessibilityServesAsTitleForUIElements() -> [AnyObject]?
   @available(OSX 10.10, *)
-  func setAccessibilityServesAsTitleForUIElements(accessibilityServesAsTitleForUIElements: [AnyObject]?)
+  func setAccessibilityServesAsTitleFor(uiElements accessibilityServesAsTitleForUIElements: [AnyObject]?)
```

Comment:
This is existing code in the Clang importer that suppresses the properties within these accessibility declarations. I don't recall the reasons off-hand.I'm not sure what you think is mangled, the name is "setAccessibilityServicesAsTitleFor(uiElements:)", which follows the proposed guideline AFAICT.